src/pacow/cowsay.clj

```diff
@@ -2,8 +2,22 @@
   (:require [clojure.java.shell :refer [sh]]
             [environ.core :refer [env]]))
 
-(def bin (env :cowsay-bin "cowsay"))
+(def cowsay (env :cowsay-bin "cowsay"))
 
-(defn moo [m]
-  (let [out (:out (sh bin m))]
-    (str "```" out "```")))
+(defn slack-out [text]
+  (str "```" text "```"))
+
+(defn animalsay [animal message]
+  (:out (sh cowsay "-f" animal message)))
+
+(defn moo [message]
+  (:out (sh cowsay message)))
+
+(defn handle-cowsay [text]
+  (let [[_ animal message] (re-find #"^\-f (.*?) (.*)" text)]
```

Comment:
I think we should change `-f`. That's the `cowsay` argument, sure, but we can make it something more intuitive like `-animal`